### ISSUE TITLE
File links not correctly decoded

### ISSUE THREAD

**Issue by kevincox** [contour-terminal/contour#1199]
### Contour Terminal version

Contour Terminal Emulator 0.3.1-2022-05-01

### Installer source

something else (please specify below)

### Operating System

nixos-unstable

### Architecture

x86-64

### Steps to reproduce

```
% echo foo >'some file'
% ls --hyperlink=auto some\ file
'some file'
```

Then CTRL+Click the link.

### Expected Behavior

The file opens in my text editor.

### Actual Behavior

Nothing happens and there is a log message.

> gio: file:///tmp/some%2520file: Error when getting information for file “/tmp/some%20file”: No such file or directory

**Comment by christianparpart** [comment]
Hello @kevincox,

many thanks for using Contour, and sorry for such a basic bug.

We need to URL-decode the passed value. Good catch. 

EDIT: `Contour Terminal Emulator 0.3.1-2022-05-01` I just noticed you are on NixOS and using an older package. We are currently on 0.3.12 as latest stable release. It might totally be still the case that we're having that bug there not fixed yet, too.

@Yaraslaut @Utkarsh-khambra NB: It might make sense to have that fixed on the 0.3.x branch as well as on master branch, so we can release a bugfix only 0.3 release.

**Comment by Utkarsh-khambra** [comment]
I'll take a look.
